Revert-tvOS/Sources/HomeViewController.swift

```diff
@@ -4,49 +4,59 @@
-      configureCell: self.dynamicType.configureCell,
-      cellIdentifier: Storyboards.Cell.Home
-    )
-
-    super.init(coder: aDecoder)
-  }
-  
   override func viewDidLoad() {
     super.viewDidLoad()
     
-    self.tableView.dataSource = self.dataSource
+    self.tableView.dataSource = self
     self.tableView.registerNib(UINib(nibName: Storyboards.Cell.Home, bundle: nil), forCellReuseIdentifier: Storyboards.Cell.Home)    
   }
   
   override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
     super.prepareForSegue(segue, sender: sender)
     
-    if let destinationViewController = segue.destinationTopViewController as? SettableHomeItem {
+    if let destinationViewController = segue.destinationTopViewController as? SettableHomeCollectionGroup {
       guard let indexPath = sender as? NSIndexPath else {
-        fatalError("`SettableHomeItem` requires `indexPath` to be sent as the sender.")
+        fatalError("`SettableHomeCollectionGroup` requires `indexPath` to be sent as the sender.")
+      }
+
+      guard let groupTitle = self.collection[indexPath.row].title else {
+        fatalError("`SettableHomeCollectionGroup` requires a named group.")
       }
-      
-      destinationViewController.item = self.collection[indexPath]
+
+      destinationViewController.collectionGroup = groupTitle
     }
   }
   
   //MARK: - Private
   
   private let collection = CollectableCollection<HomeItem>(items: .Home)
-  private let dataSource: DataSource<HomeItem, HomeCell>
 }
 
 extension HomeViewController {
   override func tableView(tableView: UITableView, didUpdateFocusInContext context: UITableViewFocusUpdateContext, withAnimationCoordinator coordinator: UIFocusAnimationCoordinator) {
-    if let nextFocusedIndexPath = context.nextFocusedIndexPath {
-      let item = self.collection[nextFocusedIndexPath]
-      self.performSegueWithIdentifier(item.segueIdentifier, sender: nextFocusedIndexPath)
-      
-      self.tableView.selectRowAtIndexPath(nextFocusedIndexPath, animated: true, scrollPosition: .None)
+    guard let nextFocusedIndexPath = context.nextFocusedIndexPath where nextFocusedIndexPath != context.previouslyFocusedIndexPath else {
+      // We don't want to perform unnecessary segues when moving back from the detail view.
+      // This also helps the detail collection view to remember the last focused item.
+
+      return
     }
+
+    self.performSegueWithIdentifier(Storyboards.Segue.DetailsCollectionView, sender: nextFocusedIndexPath)
+
+    self.tableView.selectRowAtIndexPath(nextFocusedIndexPath, animated: true, scrollPosition: .None)
+  }
+
+  override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return self.collection.count
+  }
+
+  override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+    guard let cell = tableView.dequeueReusableCellWithIdentifier(Storyboards.Cell.Home) as? HomeCell else {
```

Comment:
I'll create a card there. There's some other stuff to tackle first, so it's better to have this recorded, yep.